layouts/index.html

```diff
@@ -201,9 +199,8 @@ <h5>Archivists</h5>
             </div>
             <h5>Service providers</h5>
             <p>
-             Does your company deliver large amounts of data to users? IPFS
-             provides secure content delivery using a
-              peer-to-peer approach that could save you millions in bandwidth.
+             Providing large amounts of data to users? IPFS
+             offers secure, peer-to-peer content delivery &mdash; an approach that could save you millions in bandwidth.
```

Comment:
Should fix the end while here: "bandwidth costs"

Reply:
Done.